### ISSUE TITLE
Make the build file less repetitive

### ISSUE THREAD

**Issue by propensive** [propensive/rapture#86]
The build file repeats a lot of boilerplate for every single project. It also has four lines listing every project, which are unfriendly to merges. It would be nice to have each project defined just by a single case class.

**Comment by mielientiev** [comment]
@propensive I think it was also completed?)

**Comment by propensive** [comment]
Not really... the build file still has about ten lines for each module (of which there are about 40).
